scripts/prepare-for-platform.sh

```diff
@@ -31,17 +41,63 @@ fi
 
 scripts/run-environment-check.sh $1
 
-echo "Creating link: package.json -> ${PLATFORM_FOLDER}/package.json.orig"
-ln -sf ${PLATFORM_FOLDER}/package.json.orig package.json
+if [ ! -f .babelrc] || [ $(readlink .babelrc) != "${PLATFORM_FOLDER}/.babelrc" ]; then
+  echo "Creating link: package.json -> ${PLATFORM_FOLDER}/package.json.orig"
+  ln -sf ${PLATFORM_FOLDER}/package.json.orig package.json
 
-echo "Creating link: yarn.lock -> ${PLATFORM_FOLDER}/yarn.lock"
-ln -sf ${PLATFORM_FOLDER}/yarn.lock yarn.lock
+  echo "Creating link: yarn.lock -> ${PLATFORM_FOLDER}/yarn.lock"
+  ln -sf ${PLATFORM_FOLDER}/yarn.lock yarn.lock
 
-echo "Creating link: VERSION -> ${PLATFORM_FOLDER}/VERSION"
-ln -sf ${PLATFORM_FOLDER}/VERSION VERSION
+  echo "Creating link: VERSION -> ${PLATFORM_FOLDER}/VERSION"
+  ln -sf ${PLATFORM_FOLDER}/VERSION VERSION
 
-echo "Creating link: .babelrc -> ${PLATFORM_FOLDER}/.babelrc"
-ln -sf ${PLATFORM_FOLDER}/.babelrc .babelrc
+  echo "Creating link: .babelrc -> ${PLATFORM_FOLDER}/.babelrc"
+  ln -sf ${PLATFORM_FOLDER}/.babelrc .babelrc
 
+  yarn install --frozen-lockfile
+fi
+
+if [ "$1" == 'ios' ] || [ "$1" == 'android' ]; then
+  [ "$1" == 'android' ] && outputPath=$STATUS_GO_DRO_ARCH || outputPath=$STATUS_GO_IOS_ARCH
+  if [ -f "$outputPath" ]; then
+	  echo "status-go artifact already downloaded!"
+  else
+    echo "Downloading status-go artifact from DigitalOcean Bucket"
```

Comment:
Maybe we should make this into another impure nix package definition?
I can try doing a separate PR that does that. I want to learn a bit more about nix.

Reply:
Yeah, it's definitely the next thing to do, since it's the last remaining external artifact we're downloading into the repo.